source/includes/administration/_billing_providers.md

```diff
@@ -106,6 +106,126 @@ Provider | Attributes | &nbsp;
 ---- | ---- | -----------
 all | `creditCards`<br/>*string* | The type of credit card accepted. This is a comma separated field.
```

Comment:
This section should include all the possible attributes required by chase as well. I would move the create and update just before it as well.

Reply:
I would be nice to have the possible credit card attribute values

Reply:
da4d7b6